internal/pkg/skuba/upgrade/node/versions.go

```diff
@@ -67,27 +67,28 @@ func (nviu NodeVersionInfoUpdate) IsUpdated() bool {
 
 func (nviu NodeVersionInfoUpdate) IsFirstControlPlaneNodeToBeUpgraded() bool {
 	isControlPlane := nviu.Current.IsControlPlane()
-	currentClusterVersion, _ := kubeadm.GetCurrentClusterVersion()
+	client, _ := kubernetes.GetAdminClientSet()
```

Comment:
We should bubble this error up instead of ignore. This function return signature will be changed to `(bool, error)` in any case in: https://github.com/SUSE/skuba/pull/662#discussion_r319194135

Reply:
Fixed.